devstack/settings

```diff
@@ -0,0 +1,17 @@
+# Devstack settings
+BNP_DIR=$DEST/baremetal-network-provisioning
+BNP_ML2_CONF_HP_FILE=/etc/neutron/plugins/ml2/ml2_conf_hp.ini
+BNP_ENTRY_POINT_FILE=/opt/stack/neutron/neutron.egg-info/entry_points.txt
+SNMP_TIMEOUT=3
+SNMP_RETRIES=5
```

Comment:
it should be SNMP_RETRIES=${SNMP_RETRIES:-5}
https://github.com/openstack/networking-vsphere/blob/master/devstack/settings#L53